### PR TITLE
Revised Individual element

### PR DESCRIPTION
**Pull request by pnrobinson**
I revised the documentation for Individual and Age. Please check!

### CODE REVIEW

File: docs/individual.rst [hunk 1]
```diff
@@ -2,34 +2,38 @@
...
+   OntologyClass taxonomy = 8;
   }
 
+
+id
+==
+This element is the primary identifier for the individual. The contents of the element are context dependent, and will
+be determined by the application. For instance, if the Phenopacket is being used to represent a case study about
+an individual with some genetic disease, the individual may be referred to in that study by their position in
+the pedigree, e.g., III:2 for the second person in the third generation. In this case, id would be set to ``III:2``.
```
Comment (julesjacobsen) [inline docs/individual.rst]: Shouldn't the id in the Individual be the same as that used in the Pedigree, not the position in the pedigree?  e.g. Individual.id == Pedigree.Person.individual_id. This *could* be the position in the pedigree, but this is not mandated.

**Comment by julesjacobsen** [comment]
The ontology class example in the doc also uses the incorrect HPO onset class to define an age range.

**Comment by pnrobinson** [comment]
@julesjacobsen thanks for picking that up, I must be blind. I revised the text. I am uncertain if this comment is correct: In FHIR, age data is is represented as a UCUM measurement - http://unitsofmeasure.org/trac/

If so, I think it needs more explanation.

**Comment by julesjacobsen** [comment]
Ah yeah, the trac link is a bit useless. This is probably a more relevant link for the time units: http://unitsofmeasure.org/ucum.html#iso1000

**Comment by pnrobinson** [comment]
I have added this
```
If a ``Pedigree`` element is used, it is essential that the ``id`` used here matches the individual_id of 
the ``Pedigree`` element. 
```
I think we will need to add a page about the Pedigree element as well. 
It would be nice if we could make these pages be indented in the table of contents. I will open a new issue about the Pedigree element.

**Comment by julesjacobsen** [comment]
Does adding a ```.. toctree::``` directive to subpages produce a nested TOC?

**Comment by pnrobinson** [comment]
@julesjacobsen can we merge this? I think all of the issues are settled. Unsure what the conflict is above.

**Comment by julesjacobsen** [comment]
You needed to merge the master branch back into this one as I had pushed some changes which now conflicted with this branch.

**Pull request merged**